### ISSUE TITLE
Mic not registering on Xbox - recognizes input

### ISSUE THREAD

**Issue by athornemann** [UWPCommunity/Quarrel#512]
First off, Thank you for this app!

I can hear voice from others in channels, but no sound from my end. The app recognizes the input.

Doesn’t work with either quarrel or quarrel insider

Have:
Xbox one x
Tried with: hyperX alpha and Xbox headphones that came with purchase

**Comment by athornemann** [comment]
I now see that my name is sort of greyed out compared to the others

**Comment by Waddle-Doo** [comment]
Xbox Guide -> Audio Settings -> Headset Mic [ON]


Make sure your mic is on, then check back!

**Comment by athornemann** [comment]
Hi again, 

I did that and no luck i reinstalled the app and uninstalled again.

In the third round I was prompted with a notication that the app needed access to the microphone (though I had checked the give access box on all installs)

In summary, it works now. Thank you very much for a great app! 

A suggestion though: a volume control.. when playing games and using quarrel, the game sounds can be incredibly loud and the voice chat difficult to hear and I haven’t found a way to master volume the two

Again, thanks - and have a great day
